MissingWiki.php

```diff
@@ -6,70 +6,88 @@
-                    height: 100%;
-                    margin: 0;
-                    padding: 0;
-                    font-family: sans-serif;
-                }
-                html {
-                    font-size: 100%;
-                }
-                body {
-                    background-color: hsl(0, 0%, 96%);
-                }
-                h1, h2 {
-                    margin-bottom: .6em !important;
-                }
-                h1 {
-                    font-size: 188%;
-                }
-                h1, h2, h3, h4, h5, h6 {
-                    color: hsl(0, 0%, 0%);
-                    background: none;
-                    font-weight: normal;
-                    margin: 0;
-                    overflow: hidden;
-                    padding-top: .5em;
-                    padding-bottom: .17em;
-                    border-bottom: 1px solid hsl(0, 0%, 67%);
-                }
-                p {
-                    margin: .4em 0 .5em 0;
-                }
-            </style>
-        </head>
-        <body>
-            <div style="text-align: center;">
-                <h1>404 Wiki Not Found</h1>
-                <p>The wiki you wanted to visit does not exist. Please be sure you typed the URL correctly.</p>
-                <p>You can find all of our existing wikis <a href="https://meta.miraheze.org/wiki/Special:WikiDiscover">here</a>.</p>
-                <p>If you think there's a technical problem, please <a href="https://meta.miraheze.org/wiki/Help_center">contact the system administrators</a> and provide the following details:</p>
-                <p style="align:center;">
-                    <table style="font-style: italic; font-size: 14px;" align="center">
-                        <tbody>
-                            <tr>
-                                <td>Host: {$_SERVER['HTTP_HOST']}</td>
-                                <td>Visitor IP: {$_SERVER['REMOTE_ADDR']}</td>
-                            </tr>
-                            <tr>
-                                <td>Request URL: {$requestURL}</td>
-                                <td>Timestamp: {$fullTimestamp}</td>
-                            </tr>
-                        </tbody>
-                    </table>
-                </p> 
-            </div>
-            <div style="float: right; padding-right: 1em;">
-                <a href="https://meta.miraheze.org/wiki/Miraheze">
-                    <img src="https://static.miraheze.org/metawiki/7/7e/Powered_by_Miraheze.png" alt="Powered by Miraheze" />
-                </a>
-            </div>
-        </body>
-    </html>
+<html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta name="description" content="Wiki not Found">
+<title>Wiki not Found</title>
+<!-- Bootstrap core CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
We are not going to use a external server for this. We will have to add bootstrap into the mw-config repo same for below.